exporters/metric/cortex/config.go

```diff
@@ -97,6 +109,9 @@ func (c *Config) Validate() error {
 	if c.PushInterval == 0 {
 		c.PushInterval = 10 * time.Second
 	}
+	if c.Quantiles == nil {
+		c.Quantiles = []float64{0, 0.25, 0.5, 0.75, 1}
```

Comment:
What motivated this default?

Commonly you see something weighted to help view the tail end. E.g.

```suggestion
		c.Quantiles = []float64{0.5, 0.9, 0.95, 0.99}
```

Reply:
Given the conversion process includes a `*_min` and `*_max` metric event I think avoiding the `0` and `1` quantiles by default is a good idea.

Reply:
I didn't know what to pick, so I chose some arbitrary values instead. I'll change it to your suggestion!